src/commands.rs

```diff
@@ -1314,6 +1315,25 @@ struct ObslogArgs {
     diff_format: DiffFormatArgs,
 }
 
+/// Compare the changes of two commits
+///
+/// This excludes changes from other commits by temporarily rebasing `--to` onto
+/// `--from`'s parents. If you wish to compare the same change across versions,
+/// consider `jj obslog -p` instead.
+#[derive(clap::Args, Clone, Debug)]
+struct InterdiffArgs {
+    /// Show changes from this revision
+    #[clap(long)]
+    from: Option<String>,
+    /// Show changes to this revision
+    #[clap(long)]
+    to: Option<String>,
```

Comment:
We want at least one of them to be provided. You can copy the `ArgGroup` from the `MoveArgs` (a bit down from here)